elements/pl-symbolic-input/pl-symbolic-input.mustache

```diff
@@ -67,21 +85,29 @@ ${{a_tru}}$
 {{/answer}}
 
 {{#format}}
-Your answer must be a symbolic expression. All numbers must be rational - so, <code>1/2</code> instead of <code>0.5</code>.
-{{^allow_complex}}Complex numbers are not allowed.{{/allow_complex}}
-<hr>
+<p>Your answer must be a symbolic expression. All numbers must be rational - so, <code>1/2</code> instead of <code>0.5</code>.
+{{^allow_complex}}Complex numbers are not allowed.{{/allow_complex}}</p>
+    
 <h5><small class='text-uppercase'>Allowable Operators</small></h5>
 <pre>{{operators}}</pre>
-<p>All operations must be explicit - examples of valid expressions are <code>cos(1)</code> and <code>2*(3)</code>. Examples of invalid expressions are <code>cos 1</code> and <code>2(3)</code>. Note that either <code>^</code> or <code>**</code> can be used for exponentiation.</p>
-<hr>
+<p>All operations must be explicit:</p>
+
+<p> Example <i>valid</i> inputs: <code class="user-output">2*3</code> &nbsp; <code class="user-output">cos(1)</code> </p>
+<p> Example <i>invalid</i> inputs: <code class="user-output">2(3)</code> &nbsp; <code class="user-output">cos 1</code> </p>
+
+<p>Note that either <code>^</code> or <code>**</code> can be used for exponentiation.</p>
 <h5><small class='text-uppercase'>Allowable Constants</small></h5>
 <pre>{{constants}}</pre>{{#variables}}
 {{#allow_complex}}<p>Either <code>i</code> or <code>j</code> can be used to denote the imaginary unit. Examples of valid complex numbers are <code>1 + 2 * i</code> and <code>-3 * j</code>.</p>{{/allow_complex}}
-<hr>
 <h5><small class='text-uppercase'>Allowable Variables</small></h5>
 <pre>{{variables}}</pre>{{/variables}}
 {{/format}}
 
 {{#shortformat}}
 symbolic expression
 {{/shortformat}}
+
+{{#format_error}}
+    <hr>
+    Invalid format: {{{ format_string }}}
```

Comment:
The "Invalid format" here feels weird, given that this is the _correct format_ section. Also we already have an error message above, so it doesn't feel like we need another one.
```suggestion
    {{{ format_string }}}
```